moralization/spacy_model.py

```diff
@@ -0,0 +1,271 @@
+import spacy
+from spacy.tokens import DocBin
+from spacy import displacy
+
+from moralization.input_data import InputOutput
+from moralization.utils import is_interactive
+
+import os
+import pathlib
+from pathlib import Path
+
+from spacy.cli.init_config import fill_config
+from sklearn.model_selection import train_test_split
+from tempfile import mkdtemp
+
+
+class Spacy_Setup:
```

Comment:
class names should be in CamelCase, that means no underscore (underscores for function or method names) -> SpacySetup